### PR TITLE
Fixed wrong version of symfony with composer install

### PR DESCRIPTION
**Pull request by Nek-**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | >2.7 |
| Fixed tickets | https://github.com/symfony/symfony-docs/issues/5627#issuecomment-136015259 |

### CODE REVIEW

**Comment by javiereguiluz** [comment]
:+1: thanks @Nek- for keeping us always updated :)

File: book/installation.rst [hunk 1]
```diff
@@ -90,13 +90,12 @@ optional second argument of the ``new`` command:
 .. code-block:: bash
 
     # use the most recent version in any Symfony branch
-    $ symfony new my_project_name 2.3
```
Comment (xabbuh) [inline book/installation.rst]: Why do we remove 2.3, but keep 2.6?

Comment (Nek-) [inline book/installation.rst]: To keep LTS and non-LTS version (as in the last example).

**Comment by WouterJ** [comment]
Thanks @Nek-, I like this!

**Pull request merged**